src/flip.jl

```diff
@@ -149,8 +148,10 @@ every score at each step of the chain.
                       function is satisfied. BEWARE - this can create
                       infinite loops if the acceptance function is never
                       satisfied!
+- progress_bar        If this is true, a progress bar will be printed to stdout.
 """
-function flip_chain(
+function flip_chain_iter end  # this is a workaround
```

Comment:
...for docs, right? (specify?)

Reply:
Ah right, this is the only workaround that allow us to use `ResumableFunctions.jl` (https://github.com/BenLauwens/ResumableFunctions.jl/issues/45)